bcda/models/models.go

```diff
@@ -322,14 +322,13 @@ func (aco *ACO) GetNewAndExistingBeneficiaries(includeSuppressed bool, since str
 		}
 
 		// this is used to get unique ids for de-duplicating MBIs that are listed multiple times in the CCLF8 file
-		var uniqueIds []int64
-		db.Raw("SELECT id FROM ( SELECT max(id) as id, mbi FROM cclf_beneficiaries WHERE file_id = ? GROUP BY mbi ) as id", cclfFileNew.ID).Pluck("id", &uniqueIds)
+		uniqueIdQuery := fmt.Sprintf("SELECT id FROM ( SELECT max(id) as id, mbi FROM cclf_beneficiaries where file_id = %d GROUP BY mbi ) as id", cclfFileNew.ID)
 
 		// Populate new beneficiaries collection
 		if suppressedMBIs != nil {
-			err = db.Not("mbi", suppressedMBIs).Not("mbi", cclfBeneficiariesOld).Where("id IN (?)", uniqueIds).Find(&newBeneficiaries, "file_id = ?", cclfFileNew.ID).Error
+			err = db.Not("mbi", suppressedMBIs).Not("mbi", cclfBeneficiariesOld).Where("id IN (?)", db.Raw(uniqueIdQuery).SubQuery()).Find(&newBeneficiaries, "file_id = ?", cclfFileNew.ID).Error
```

Comment:
Should we also consider adding this protection in the following spots?

1. Not("mbi", suppressedMBIs)
2. Not("mbi", cclfBeneficiariesOld)
3. Where("mbi IN (?)", cclfBeneficiariesOld)

Reply:
Absolutely necessary.   Great catch @mtrang1263 !

Reply:
The `suppressedMBIs` arrays are bound here, so I don't think there is a pressing need to refactor those.  If you think strongly, we can add a separate ticket to address.  
I definitely agree that the `cclfBeneficiariesOld` arrays should be refactored to use a SubQuery.  Great catch @mtrang1263 !  I have updated in latest commit.